File: code-samples/social-messaging/code-snippets/reply-message.php
<?php
reply_message("A-Valid-Message-Id");

/*
  Reply to a message
*/
function reply_message($messageId){
  global $platform;
  try {
    $bodyParams = array (
      'inReplyToContentId' => $messageId,
      'body' => "Thank you for your message! PHP",
      'autoSubmitted' => True,
      'public' => False
    );

    $endpoint = "/cx/social-messaging/v1/contents";

    $resp = $platform->post($endpoint, $bodyParams);
    $jsonObj = $resp->json();
    print_r (json_encode($jsonObj, JSON_PRETTY_PRINT) . PHP_EOL);
  } catch (\RingCentral\SDK\Http\ApiException $e) {
    print_r ("Unable to reply to this messages. Error message:" . $e->getMessage() . PHP_EOL);
  }
}
?>

File: code-samples/social-messaging/code-snippets/reply-message.py
import json
from ringcentral import SDK

# Read the current authenticated user's message store.
def reply_message(messageId):
    try:
        bodyParams = {
            'inReplyToContentId': messageId,
            'body': "Thank you for your message! Python",
            'autoSubmitted': True,
            'public': False
          }
        endpoint = "/cx/social-messaging/v1/contents"
        resp = platform.post(endpoint, bodyParams)
        jsonObj = resp.json_dict()
        print(json.dumps(jsonObj, indent=2, sort_keys=True))
    except Exception as e:
        print ("Unable to reply to this message. Error message: " + str(e))


# Authenticate a user using a personal JWT token
def login():
    try:
      platform.login( jwt= "RC_USER_JWT" )
      reply_message("A-Valid-Message-Id")
    except Exception as e:
      print ("Unable to authenticate to platform. Check credentials." + str(e))
